drape_frontend/my_position_controller.cpp

```diff
@@ -178,6 +178,12 @@ void MyPositionController::CorrectScalePoint(m2::PointD & pt1, m2::PointD & pt2)
   }
 }
 
+void MyPositionController::CorrectGlobalScalePoint(m2::PointD & pt) const
```

Comment:
Confusing name for the method. Would it be easier to return point and name it like GetXXX?